Update `Publish your own post` (#27)
iScsc/blog.iscsc.fr: src/content/posts/publish-your-own-post/index.md

Also as stated later, if you're struggling with something, you can contact me on Discord: `ctmbl` or open an [Issue](https://github.com/iScsc/blog.iscsc.fr/issues) on the GitHub repository.

> **DISCLAIMER**: This article has been written for `git`/GitHub **beginners**, to publish through the GitHub's web interface.

> > If you're used to `git` cloning and GitHub's forking and PR mechanisms you can create the Pull Request has you're used to!

Reviewer: ```suggestion
> If you're used to `git` cloning and GitHub's forking and PR mechanisms you can create the Pull Request as you're used to!
```
A small spelling mistake

Reviewer: The rest seems good to me!

Now let's create the associated **Pull Request** to finally share your article with the world!

#### 3.4.A- Create PR directly...

> > **NOTE**: if you don't end up on the same webpage than me, skip this **subsection** and go to `OR from the repo's page` (3.4.B subsection).

Reviewer 2: ```suggestion
> **NOTE**: if you don't end up on the same webpage than me, skip this **subsection** and go to `OR from the repo's page` (3.4.B).
```
or `(subsection 3.4.B)` maybe but tks good tips!

> > **NOTE 2**: if you're interested in why we're doing this, `git` mechanisms (branch, ...) and `GitHub` ones (repos, Pull Requests, Forks), I should right soon post an article on the [blog](https://iscsc.fr)

Reviewer 2: ```suggestion
> **NOTE 2**: if you're interested in why we're doing this, `git` mechanisms (branch, ...) and `GitHub` ones (repos, Pull Requests, Forks), I should write soon an article on the [blog](https://iscsc.fr)
```
oops ~~right~~ instead of "write" :facepalm: 
tks!
